extensions/amp-ad-exit/0.1/config.js

```diff
@@ -60,6 +61,21 @@ export let VariableDef;
  */
 export let VariablesDef;
 
+/**
+ * Supported Behaviors:
+ *  -- clickTarget -- Specifies where to open the click, defaults to '_blank'
```

Comment:
I'd clarify that the choice is either "_top", meaning "_top", or any other value, meaning "_blank". Also that this is merely a preference if the behavior is possible in the environment (e.g. a CCT might always navigate top, or a mobile app might always open the system browser).

Please also describe this in amp-ad-exit.md